src/Elliptic/Systems/GetSourcesComputer.hpp

```diff
@@ -5,6 +5,8 @@
 
 #include <type_traits>
 
+#include "Utilities/TMPL.hpp"
```

Comment:
Could you please also add this diff as well:

```diff
diff --git a/src/Elliptic/Systems/CMakeLists.txt b/src/Elliptic/Systems/CMakeLists.txt
index 2e0108654..fb499b48f 100644
--- a/src/Elliptic/Systems/CMakeLists.txt
+++ b/src/Elliptic/Systems/CMakeLists.txt
@@ -12,6 +12,12 @@ spectre_target_headers(
   GetSourcesComputer.hpp
   )
 
+target_link_libraries(
+  ${LIBRARY}
+  INTERFACE
+  Utilities
+  )
+
 add_subdirectory(Elasticity)
 add_subdirectory(Poisson)
 add_subdirectory(Xcts)
```

Reply:
Thanks, I have squashed it directly into the commit.